### ISSUE TITLE
remove the invasive checkbox

### ISSUE THREAD

**Issue by wbazant** [falling-fruit/falling-fruit-web#560]
This view for example has small clusters:

https://beta.fallingfruit.org/map/@39.9654518,-105.4882976,13z

Toggle "invasive species only" - nothing happens. Then zoom in, and they disappear.

@ezwelty I remember we discussed the option of removing the invasive checkbox, what do you think? Shall we just not have that checkbox?

Edit: I've found https://github.com/falling-fruit/falling-fruit-web/issues/383#issuecomment-2142951728 that confirms the data's not there with the suggestion to hide it for now. I'll change this to an 'enhancement' with a proposed resolution to remove.

**Comment by lattaai1** [comment]
Hello, I tried to fix the issue

This is what I did:
I have removed the invasive species filter from the Filter component and related Redux slices. This change addresses the issue where toggling the invasive species filter had no effect and caused unexpected behavior when zooming in. The filter was removed from the UI and the corresponding state and actions in the Redux store.

I also created pull request: https://github.com/falling-fruit/falling-fruit-web/pull/561


> [!CAUTION]  
> Disclaimer: This fix was created by Latta AI and you should never merge before you check the correctness of generated code!
> 
> The fix provided by Latta AI might not be complete and it can serve as an inspiration.**

---
This bug was fixed for free by Latta AI - https://latta.ai/ourmission

If you no longer want Latta AI to attempt fixing issues on your repository, you can block this account.

**Comment by ezwelty** [comment]
@wbazant Indeed, it isn't available at the cluster-level. So we could remove it, but I might suggest we disable it until a minimum zoom is reached (much like the add location button). This has the advantage of keeping the feature visible (we have a neat text to explain the invasivore concept on the live site), and other filters will probably need to be dealt with in this same way (e.g. by ripeness: https://github.com/falling-fruit/falling-fruit-api/issues/25, by access: https://github.com/falling-fruit/falling-fruit-api/issues/23).

**Comment by wbazant** [comment]
That does address the 'no data at cluster level' issue - but furthermore, I don't like the checkbox ;). I didn't want to write a big rant against it but in essence, it seemed to me like it's there to educate users about 'eating the weeds' but I think it doesn't support much meaningful UX. The user can try toggling it, and might see a few 'edible invasive' types on their map, but most likely they'll get something of limited interest, for example this is Boulder, CO:
![Screenshot from 2024-11-20 19-20-32](https://github.com/user-attachments/assets/c514887b-b9c3-4c3f-9495-3ceb92bbd926)

![Screenshot from 2024-11-20 19-19-15](https://github.com/user-attachments/assets/f97e2652-d84a-420b-8d70-1442b73f1513)

An extra reason why I don't like the checkbox is that there's no data for my location ;). I remember the reason it's only in the US is because you've used USDA state-level data in some way.

Maybe a better way to support the same functionality - tell people to eat the weeds just as persuasively, show them invasive edibles on the map - would be a little landing page for eating the weeds, with a link to a pre-configured type filter: it would have a title like "Invasive edibles CO", a description, a default start, and a collection of types that it contains. I've only had the idea five minutes ago but how does reinventing this part of the site sound and am I missing something about how users are interacting with the feature now?
